src/status_im/tests/integration_test/core_test.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.tests.integration-test.core-test
```

Comment:
In terms of structure, it's odd that our tests live inside `src/status-im`. I think we should move them to `<root>/test/integration_test`, a common structure in many Clojure projects.

Reply:
Just a minor detail @J-Son89, but by `<root>` here I meant the root of the project, where we already have a `test` directory, and where many Clojure projects put their tests. But where you moved them is already better 👍🏼

Reply:
Oh yeah but do we not need some details within src for the compiler?

Reply:
And perhaps other tooling?

Reply:
Shadow-cljs just needs to be told which directories it should look for Clojure files. From there, it can compile any clojure file requiring any namespace it found. In our case, we currently have `["src" "test/cljs"]` in `:source-paths`. I believe `test/cljs` is a leftover. If we wanted shadow to see integration tests in `status-mobile/test/integration-test`, we would just change `:source-paths` to `["src" "test"]`.

Then change the namespaces and update one or two lines in the Makefile that currently have the `status-im.*` part hardcoded. From there, we would have moved tests outside the `src` directory! We can do this in a separate PR if we agree.

Reply:
sure, we can do this in a follow up. It would be nice to get the Contract Tests started so we can add some more tests sooner.